paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/GooglePayTest.kt

```diff
@@ -132,78 +129,6 @@ internal class GooglePayTest {
         }
     }
 
-    @Test
-    fun googlePayFailsOnGooglePaySheetFailure() {
-        val flowControllerFailureResult = CountDownLatch(1)
-
-        runGooglePayFlowTest(
-            paymentResultCallback = {
-                when (integrationType) {
-                    ProductIntegrationType.PaymentSheet -> expectNoResult(it)
-                    ProductIntegrationType.FlowController -> {
-                        assertFailed(it)
-                        flowControllerFailureResult.countDown()
-                    }
-                }
-            }
-        ) { scenario ->
-            intendingGooglePayToBeLaunched(
-                GooglePayPaymentMethodLauncher.Result.Failed(
-                    error = IllegalStateException("An error occurred!"),
-                    errorCode = GooglePayPaymentMethodLauncher.INTERNAL_ERROR
-                )
-            )
-
-            scenario.confirm()
-
-            intendedGooglePayToBeLaunched()
-
-            when (integrationType) {
-                ProductIntegrationType.PaymentSheet -> {
-                    waitForErrorMessage("An internal error occurred.")
-                    scenario.markTestSucceeded()
-                }
-                ProductIntegrationType.FlowController -> flowControllerFailureResult.await()
-            }
-        }
-    }
-
-    @Test
-    fun googlePayFailsOnPaymentFailure() {
-        val flowControllerFailureResult = CountDownLatch(1)
-
-        runGooglePayFlowTest(
-            paymentResultCallback = {
-                when (integrationType) {
-                    ProductIntegrationType.PaymentSheet -> expectNoResult(it)
-                    ProductIntegrationType.FlowController -> {
-                        assertFailed(it)
-                        flowControllerFailureResult.countDown()
-                    }
-                }
-            }
-        ) { scenario ->
-            intendingGooglePayToBeLaunched(
-                GooglePayPaymentMethodLauncher.Result.Failed(
-                    error = IllegalStateException("An error occurred!"),
-                    errorCode = GooglePayPaymentMethodLauncher.INTERNAL_ERROR
-                )
-            )
-
-            scenario.confirm()
-
-            intendedGooglePayToBeLaunched()
-
-            when (integrationType) {
-                ProductIntegrationType.PaymentSheet -> {
-                    waitForErrorMessage("An internal error occurred.")
-                    scenario.markTestSucceeded()
-                }
-                ProductIntegrationType.FlowController -> flowControllerFailureResult.await()
-            }
-        }
-    }
-
```

Comment:
Removes these emulator tests in favor of the Robolectric variants. Kept the succeeding case for coverage since this also tests with `PaymentSheet`